include/yaml-cpp/node/node.h

```diff
@@ -130,6 +132,7 @@ class YAML_CPP_API Node {
 
  private:
   bool m_isValid;
+  std::string m_invalidKey;
```

Comment:
Add a comment, that this is the string representation of an invalid key, if this node is invalid.

Reply:
Done.

Reply:
Style nit: Please capitalize the first letter of the comment, and put a period at the end.